BrokerApiNode/README.md

```diff
@@ -0,0 +1,82 @@
+# Broker APIs
+This example demostrates how to use the Streamr Broker API's from NodeJS. Node version 14/16 or higher is recommended.
+
+## Broker & SDKs
+- [Examples directory](https://github.com/streamr-dev/examples/blob/ECODR-34-broker-and-client/NodeJs/src/broker/)
+
+The Broker acts as a relayer node from it's HTTP/WS/MQTT APIs into the Streamr Network
+
+### Broker setup
+```shell
+$ npm install streamr-broker --global
+$ npm run broker:start
+```
+The [default config](https://github.com/streamr-dev/examples/blob/ECODR-34-broker-and-client/NodeJs/broker-config.json) has the HTTP, WS and MQTT plugins enabled by default to allow for SDK interaction.
+
+You can also start the broker with the apiKey `0x1c4343df92f5370208232782c373fa691c3543bdf4c40adfd406c87103b18fc2` by running:
+
+```
+$ npm run broker:start-api-key
+```
+
+In this setup you'll have to provide one of the registered apiKeys under `src/config-api-key.json` to be able to interact with the broker node. 
+
+### HTTP Endpoint
+- [API Reference](https://api-explorer.streamr.com/)
```

Comment:
The API explorer is for the mostly deprecated `core-api`. Is there anything that you reach for from the explorer?